### ISSUE TITLE
Font-size to large on windows

### ISSUE THREAD

**Issue by jhony1104** [Murmele/Gittyup#690]
On Windows 11 the font-size of everything but diffs and line-height in diffs is way to large.

![image](https://github.com/Murmele/Gittyup/assets/22909893/e69b5f2b-ae65-4e74-9361-6ee92e1247fb)

It seems to be independent of monitor scaling. (Both 100% and 250% look the same).

**Comment by Murmele** [comment]
Hi, which version did you use?

**Comment by jhony1104** [comment]
- Gittyup 1.3.0 _(current release on GitHub)_
- Windows 11 23H2 22631.3007

**Comment by Murmele** [comment]
Can you try with the development version?

**Comment by jhony1104** [comment]
Are there binaries available for the development version?

**Comment by Murmele** [comment]
Yes you can find them here: https://github.com/Murmele/Gittyup/releases/tag/development

**Comment by jhony1104** [comment]
The proble still exists with the development version

**Comment by Murmele** [comment]
I think I have now a similar problem on a linux machine. I did nit change anything, just restarted the application. Maybe it is something related

**Comment by jhony1104** [comment]
I never had the app running without the issue. My guesse is tha it is a qt issue, since i have seen similar problems with other qt apps.

**Comment by madeddy** [comment]
I have this also in LX:
![Screenshot_20240408_231105](https://github.com/Murmele/Gittyup/assets/26324301/57319ca2-5ba4-4973-a0ea-408c3b74acb6)

Version: _dev_ from 16.March 2024 -> Gittyup-1.3.0-x86_64.AppImage
OS: Kubuntu 22.04.4

Its not connected to the font or size.

**Comment by jhony1104** [comment]
At least for windows this is fixed in current development version. @madeddy please check if you can still replicate the problem.

**Comment by madeddy** [comment]
> @madeddy please check if you can still replicate the problem.

It is now with dev-v1.4.0 a bit worse than before. My sys-stats are unchanged.

The @@ lines now are half overlapped by the line below:
![Screenshot_20240429_120429](https://github.com/Murmele/Gittyup/assets/26324301/724869ce-e348-4159-b0ef-98715ff2edb1)

**Comment by mayeulk** [comment]
I can confirm on Kubuntu 24.04.3 LTS, Gittyup prod version installed today as flatpak.
Reducing the main font size in KDE's font setting improves the situation (for some parts of Gittyup only) but impacts all open application. Makes texts hard to read (too small) on high res 4k display for some tasks / applications. Tested with fonts: Noto, DejaVu Sans.

**Comment by AHSauge** [comment]
I suspect this got fixed with the upgrade to Qt6. I'm not able to reproduce this with latest master at least, which implies this likely is fixed in v2.0.0 already.
